src/client/opamArg.mli

```diff
@@ -25,15 +25,18 @@ val cli2_0: OpamCLIVersion.t
 val cli2_1: OpamCLIVersion.t
 val cli2_2: OpamCLIVersion.t
 
-(* [cli_from since] validity flag since [since], and no removal version *)
-val cli_from: OpamCLIVersion.t -> validity
+(* [cli_from ?experimental since] validity flag since [since], and no removal
+   version. If experimental is given, it is marked as is (warning and doc
+   update). *)
+val cli_from: ?experimental:bool -> OpamCLIVersion.t -> validity
```

Comment:
Do we want default cli on new flags ?

Reply:
I don't follow?

Reply:
On removed flags (`cli_between`), it is possible to specify that we want to keep them on default cli (like `opam config var`). This `~default` argument is not available for non removed flags (created with `cli_from`). 
Given the usage of default cli, i don't think that it is needed to permit non deprecated flags as in the default cli ; but wondering if i'm missing some edge cases.